From 45972f6fd8f7f726f41397d9d6f7ea5eda933298 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 23 Sep 2020 11:28:16 +0800
Subject: [PATCH] Test case for delete JustATest.uni in FrontPageStrings.uni

Description: delete JustATest.uni in the MdeModulePkg/Application/UiApp/FrontPageStrings.uni

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/FrontPageStrings.uni | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/MdeModulePkg/Application/UiApp/FrontPageStrings.uni b/MdeModulePkg/Application/UiApp/FrontPageStrings.uni
index 8662e009bd..bedce5244f 100644
--- a/MdeModulePkg/Application/UiApp/FrontPageStrings.uni
+++ b/MdeModulePkg/Application/UiApp/FrontPageStrings.uni
@@ -8,8 +8,10 @@
 //**/
 
 /=#
-
-#include "JustATest.uni"
+#langdef   en-US "FrançaisEnglish"
+#langdef   fr-FR "English"
+#langdef   en    "Standard English"
+#langdef   fr    "Standard Français"
 
 #string STR_FRONT_PAGE_TITLE           #language en-US  "Front Page"
                                        #language fr-FR  "Front Page"
-- 
2.27.0.windows.1

